docs/validate/upgrades/list-of-upgrades.md

```diff
@@ -8,40 +8,48 @@ Check the details and requirements for each mainnet and testnet upgrade.
-| [`v9.1.0`](https://github.com/evmos/evmos/releases/tag/v9.1.0)           |   ✅    |    ✅    |     ❌     |         ❌          | [6,654,000](https://www.mintscan.io/evmos/blocks/6654000)   |
-| [`v8.2.0`](https://github.com/evmos/evmos/releases/tag/v8.2.0)           |   ✅    |    ✅    |     ❌     |         ✅          | [4,888,000](https://www.mintscan.io/evmos/blocks/4888000)   |
-| [`v8.1.0`](https://github.com/evmos/evmos/releases/tag/v8.1.0)           |   ✅    |    ✅    |     ❌     |         ❌          | [4,655,500](https://www.mintscan.io/evmos/blocks/4655500)   |
-| [`v7.0.0`](https://github.com/evmos/evmos/releases/tag/v7.0.0)           |   ✅    |    ✅    |     ❌     |         ✅          | [2,476,000](https://www.mintscan.io/evmos/blocks/2476000)   |
-| [`v6.0.2`](https://github.com/evmos/evmos/releases/tag/v6.0.2)           |   ✅    |    ❌    |     ❌     |         ✅          |                                                             |
-| [`v6.0.1`](https://github.com/evmos/evmos/releases/tag/v6.0.1)           |   ✅    |    ✅    |     ❌     |         ❌          | [1,042,000](https://www.mintscan.io/evmos/blocks/1042000)   |
-| [`v5.0.0`](https://github.com/evmos/evmos/releases/tag/v5.0.0)           |   ✅    |    ✅    |     ❌     |         ❌          | [837,500](https://www.mintscan.io/evmos/blocks/837500)      |
-| [`v4.0.1`](https://github.com/evmos/evmos/releases/tag/v4.0.1)           |   ❌    |    ✅    |     ❌     |         ❌          | [257,850](https://www.mintscan.io/evmos/blocks/257850)      |
-| [`v3.0.0`](https://github.com/evmos/evmos/releases/tag/v3.0.0)           |   ❌    |    ✅    |     ✅     |         ✅          | [58,701](https://www.mintscan.io/evmos/blocks/58701)        |
-| [`v2.0.1`](https://github.com/evmos/evmos/releases/tag/v2.0.1)           |   ❌    |    ❌    |     ❌     |         ❌          | [58,700](https://www.mintscan.io/evmos/blocks/58700)        |
-| [`v2.0.0`](https://github.com/evmos/evmos/releases/tag/v2.0.0)           |   ✅    |    ✅    |     ❌     |         ❌          | [58,700](https://www.mintscan.io/evmos/blocks/58700)        |
-| [`v1.0.0`](https://github.com/evmos/evmos/releases/tag/v1.0.0) (genesis) |  `N/A`  |  `N/A`   |   `N/A`   |         ❌          | [1](https://www.mintscan.io/evmos/blocks/1)                 |
+| Version                                                                  | Planned | Breaking | Data Reset | Manual Upgrade Only | Upgrade Height                                              |
+|--------------------------------------------------------------------------|:-------:|:--------:|:----------:|:-------------------:|-------------------------------------------------------------|
+| [`v16.0.2`](https://github.com/evmos/evmos/releases/tag/v16.0.2)         |    ✅    |    ✅     |     ❌      |          ❌          | [18,295,000](https://www.mintscan.io/evmos/blocks/18295000) |
```

Comment:
Shouldnt patch versions not be breaking?

Reply:
yes, thanks for the catch 🙏 

```suggestion
| [`v16.0.2`](https://github.com/evmos/evmos/releases/tag/v16.0.2)         |    ✅    |    ❌     |     ❌      |          ❌          | [18,295,000](https://www.mintscan.io/evmos/blocks/18295000) |
```